### PR TITLE
Update maintainers.md

### PR DESCRIPTION
**Pull request by AbhijeetSharma1**
Contribution update

### CODE REVIEW

**Comment by AbhijeetSharma1** [comment]
Contribution merging

**Review by axljoker03 — APPROVED** [review]

**Review by terrajobst — APPROVED** [review]

File: Documentation/contributing/maintainers.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ The following two lists summarize how maintainers will act. A more detailed desc
 
 **Maintainers won't**
 - Edit contributor comments as a general rule. GitHub provides [edit history](https://blog.github.com/changelog/2018-05-24-comment-edit-history/) which makes editing more acceptable as there is a way to see an audit log. However, contributors might still be surprised when they see statements or phrases under their name that they don't recall making. Transparency is a cornerstone of effective discourse and its absence undermines trust. And a change log doesn't fully eliminate these concerns.
+- Accept the same contribution which has been done before by someone else.
```
Comment (mairaw) [inline Documentation/contributing/maintainers.md:15]: In general, we strive to do that. @AbhijeetSharma1 is there any particular reason you want to explicitly add this language? Cases I've seen this happen is either by accident or someone who's responding to feedback or CLA agreements, so we can't merge.

Comment (terrajobst) [reply Documentation/contributing/maintainers.md:15]: My reason for approval was that it seemed aligned with our general principle of not wanting to have multiple ways in the product to do the same thing. But now I'm thinking we're generally not evaluating if the contribution was taken from somewhere else; that's what the CLA is for (i.e. the onus is on the submitter). That simply means that if there are complaints later, we might have to remove code if it wasn't owned by the person who contributed it to our repo.

However, we're generally looking for obvious cases (e.g. sources we know has incompatible licenses, such as the Linux kernel).

**Comment by mairaw** [comment]
Closing since there was no activity in this PR in months. Happy to reactivate if we get more clarity.

**Pull request closed without merging**